recipes/blake2/all/conanfile.py

```diff
@@ -0,0 +1,70 @@
+import os
+import stat
+import shutil
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+
+class blake2Conan(ConanFile):
+    name = "blake2"
+    version = "997fa5b" #BLAKE2 doesn't have versions so we use the commit hash instead
```

Comment:
Use the release date instead, not the commit

https://github.com/BLAKE2/BLAKE2/releases/tag/20160619
https://github.com/conan-io/conan-center-index/wiki/FAQ#what-version-should-packages-use-for-libraries-without-official-releases

Reply:
I'll use the release date instead of the commit, it makes more sense. Regardless, I can't use the release source because is way too old and it doesn't have the NEON sources.

Reply:
Nice